CONTRIBUTING.rst

```diff
@@ -363,21 +376,25 @@ Here are the key steps you need to go through to contribute code to `nilearn`:
 
       git checkout -b your_branch
 
-3. implement and commit your changes on this branch (don't forget to write tests!)
+3. implement changes and (optional but highly recommended) lint::
+
+      git diff HEAD | flake8 --diff
```

Comment:
I believe this should also match the suggestion from @jeromedockes

Reply:
or even better show the linting command in one place only